<ui version="4.0" >
 <class>FoliaMain</class>
 <widget class="QMainWindow" name="FoliaMain" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Folia</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <widget class="QWidget" name="dbTab" >
       <attribute name="title" >
        <string>Base de données</string>
       </attribute>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Classe:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="classListBox" >
              <property name="minimumSize" >
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="newClassButton" >
              <property name="text" >
               <string>Nouvelle classe</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget_2" >
            <property name="currentIndex" >
             <number>0</number>
            </property>
            <widget class="QWidget" name="sampleTab" >
             <attribute name="title" >
              <string>Échantillons</string>
             </attribute>
             <layout class="QHBoxLayout" >
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <layout class="QGridLayout" >
                  <property name="horizontalSpacing" >
                   <number>-1</number>
                  </property>
                  <item row="0" column="0" colspan="2" >
                   <widget class="QLabel" name="label_2" >
                    <property name="font" >
                     <font>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text" >
                     <string>Échantillons:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" colspan="2" >
                   <widget class="QListView" name="samplesList" >
                    <property name="sizePolicy" >
                     <sizepolicy vsizetype="MinimumExpanding" hsizetype="Expanding" >
                      <horstretch>2</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0" >
                   <spacer>
                    <property name="orientation" >
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" >
                     <size>
                      <width>61</width>
                      <height>24</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="2" column="1" >
                   <widget class="QToolButton" name="addSample" >
                    <property name="text" >
                     <string>Ajouter</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QGridLayout" >
                  <item row="0" column="0" >
                   <widget class="QLabel" name="label_3" >
                    <property name="text" >
                     <string>Paramètres:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" >
                   <widget class="QTableView" name="paramsTable" />
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="distTab" >
             <attribute name="title" >
              <string>Distributions</string>
             </attribute>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QTableView" name="distTable" />
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="classifyTab" >
       <attribute name="title" >
        <string>Classification</string>
       </attribute>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QGridLayout" >
            <item row="0" column="0" colspan="4" >
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="label_4" >
                <property name="font" >
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text" >
                 <string>Fichier:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="fileLabel" >
                <property name="text" >
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="selFileToClassify" >
                <property name="text" >
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0" >
             <widget class="QRadioButton" name="radioFront" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>50</width>
                <height>5</height>
               </size>
              </property>
              <property name="text" >
               <string>Face</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2" >
             <widget class="QRadioButton" name="radioBack" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Dos</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>381</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="2" >
             <widget class="QPushButton" name="classifyButton" >
              <property name="text" >
               <string>Identifier</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3" >
             <widget class="QPushButton" name="razButton" >
              <property name="text" >
               <string>Remize à zéro</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="image" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="idLabel" >
            <property name="font" >
             <font>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier" >
    <property name="title" >
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionChanger_de_base_de_donn_es" />
    <addaction name="separator" />
    <addaction name="actionQuitter" />
   </widget>
   <addaction name="menu_Fichier" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionChanger_de_base_de_donn_es" >
   <property name="text" >
    <string>Changer de base de données...</string>
   </property>
  </action>
  <action name="actionQuitter" >
   <property name="text" >
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
